pyaedt/modeler/PrimitivesNexxim.py

```diff
@@ -1058,7 +1058,7 @@ def assign_sin_excitation2ports(self, ports, settings):
 
         for prt in ports:
             arg_temp = []
-            arg_temp = ["NAME:EnabledAnalyses", ["NAME:"+str(prt), str(prt)+":=", [] ], ["NAME:VoltageSinusoidal"+str(id), str(prt)+":=", [] ] ]
+            arg_temp = ["NAME:Enabledanalyzes", ["NAME:"+str(prt), str(prt)+":=", [] ], ["NAME:VoltageSinusoidal"+str(id), str(prt)+":=", [] ] ]
```

Comment:
```suggestion
            arg_temp = ["NAME:EnabledAnalyses", ["NAME:"+str(prt), str(prt)+":=", [] ], ["NAME:VoltageSinusoidal"+str(id), str(prt)+":=", [] ] ]
```
I think this one should stay the same, since it is using the plural of analysis, which is analyses.

Reply:
Agreed. Thanks @jleonatti !